src/llmq/utils.cpp

```diff
@@ -955,10 +954,7 @@ bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, const CQuorumMana
             break;
         case Consensus::LLMQType::LLMQ_100_67:
         case Consensus::LLMQType::LLMQ_TEST_V17:
-            if (LOCK(cs_llmq_vbc); VersionBitsState(pindex, consensusParams, Consensus::DEPLOYMENT_DIP0020, llmq_versionbitscache) != ThresholdState::ACTIVE) {
-                return false;
-            }
-            break;
+            if (pindex->nHeight < consensusParams.DIP0020Height) return false;
```

Comment:
missing `break` here. because if condition is false, behaviour changed, seems so.